autobisect/bisect.py

```diff
@@ -181,6 +181,82 @@ def __init__(
 
         self.build_manager = BuildManager(config)
 
+    def _get_daily_builds(self):
+        """
+        Create build range containing one build per day
+        """
+        start = self.start.datetime + timedelta(days=1)
```

Comment:
self.start.datetime and self.end.datetime seem to be relied upon in a few places here and they doesn't seemed to be sanity checked. Maybe add `assert end > start` or whatever makes sense to `__init__()`?

Reply:
Good call.  I'm going to add this in the next PR which will make some significant changes to `Bisector.__init__()` and add a class method for handling args.